services/php-fpm-nginx/Dockerfile

```
@@ -1,59 +1,49 @@
+ENV COMPOSER_NO_INTERACTION=1 \
+    COMPOSER_DISCARD_CHANGES=1 \
+    COMPOSER_ALLOW_SUPERUSER=1 \
+    DOCROOT=/usr/share/nginx/html
+
+# Configure php-fpm to run as a service.
+COPY files/service/php-fpm/run /root/php-fpm-run
+COPY files/tugboat.ini /root/tugboat.ini
+COPY files/nginx.conf /root/nginx.conf
+COPY files/sites-enabled/* /root/sites-enabled/
+
 # Debian images by default disable apt caching, so turn it on until we finish
 # the build.
-RUN mv /etc/apt/apt.conf.d/docker-clean /etc/apt/docker-clean-disabled
 
 # sharing=locked prevents concurrent builds from accessing this cache at the
 # same time. To improve build performance, we ensure we only run apt related
 # commands in this layer.
-RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
+RUN mv /etc/apt/apt.conf.d/docker-clean /etc/apt/docker-clean-disabled && \
+    --mount=type=cache,target=/var/cache/apt,sharing=locked \
     --mount=type=cache,target=/var/lib/apt,sharing=locked \
```

```suggestion
RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
    --mount=type=cache,target=/var/lib/apt,sharing=locked \
    mv /etc/apt/apt.conf.d/docker-clean /etc/apt/docker-clean-disabled && \
```

https://docs.docker.com/engine/reference/builder/#run---mounttypecache